controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -49,12 +49,14 @@
     public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers, ServerCallHandler<ReqT, RespT> next) {
 
         Map<String, String> paramMap = new HashMap<>();
-        headers.keys().stream().forEach(key -> {
+        headers.keys().stream()
+               .filter(key -> !key.endsWith(Metadata.BINARY_HEADER_SUFFIX))
+               .forEach(key -> {
             try {
```

Comment:
The indentation seems in correct with respect to the new code.

Reply:
Fixed.